Packs/CommunityCommonScripts/Scripts/ToCase/ToCase.py

```diff
@@ -0,0 +1,11 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+value = demisto.args()['value']
+case = demisto.args().get('case', 'capital')
+if case == 'swap':
+    value = value.swapcase()
+elif case == 'capital':
+    value = value.capitalize()
+elif case == 'title':
+    value = value.title()
+return_results(value)
```

Comment:
you're using an if-elif block to check the case variable. I'd like to suggest using match-case, introduced in Python 3.10. This can simplify the code and make it cleaner and more modern.

Reply:
Done